resources/sdk/purecloudpython/templates/api.mustache

```diff
@@ -60,8 +59,6 @@ class {{classname}}(object):
                 config.api_client = ApiClient()
             self.api_client = config.api_client
 {{#operation}}
-
-    {{#isDeprecated}}@deprecated("{{operationId}} is deprecated"){{/isDeprecated}}
     def {{operationId}}(self, {{#sortParamsByRequiredFlag}}{{#allParams}}{{#required}}{{paramName}}: {{#vendorExtensions}}{{{x-dataType}}}{{/vendorExtensions}}, {{/required}}{{/allParams}}{{/sortParamsByRequiredFlag}}**kwargs) -> {{#returnType}}{{#vendorExtensions}}{{{x-returnType}}}{{/vendorExtensions}}{{/returnType}}{{^returnType}}None{{/returnType}}:
```

Comment:
if you are having problem with @deprecated. 
may be you could implement something like this and try ?

import warnings
from functools import wraps

def deprecated(func):
    @wraps(func)
    def wrapper(*args, **kwargs):
        warnings.warn(f"{{operationId}} is deprecated", DeprecationWarning)
        return func(*args, **kwargs)
    return wrapper

@deprecated
def {{operationId}}():
------

the deprecated decorator wraps the  deprecated function. When it is called, it will generate a deprecation warning  message

Reply:
Thanks Hemanth I'll try that